src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/WebServices/AddAzureMLWebServiceRegionalProperty.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.MachineLearning.Utilities;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Models;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Util;
+using Microsoft.WindowsAzure.Commands.Common;
+
+namespace Microsoft.Azure.Commands.MachineLearning.Cmdlets
+{
+    [Cmdlet(
+        VerbsCommon.Add,
+        "AzureRmMlWebServiceRegionalProperty",
+        SupportsShouldProcess = true)]
+    [OutputType(typeof(void))]
+    public class AddAzureMLWebServiceRegionalProperty : WebServicesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the resource group for the Azure ML web service.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "The name of the web service")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
```

Comment:
@NonStatic2014 I think `Name` and `ResourceGroupName` should have the `ValueFromPipelineByPropertyName` attribute so you are able to pipe the result of `Get-AzureRmMlWebService` to this cmdlet. Does that seem like a scenario that a user may use this cmdlet for?